mreg_cli/types.py

```diff
@@ -4,8 +4,10 @@
 import sys
 from typing import TYPE_CHECKING, Dict, List, Optional, Union
 
+from typing_extensions import Literal, TypeAlias  # noqa: F401
```

Comment:
This should be moved into an `if TYPE_CHECKING` block, since we can't import `typing_extensions` on runtime for RHEL 7 machines.

I.e.

```py
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from typing_extensions import Literal  # noqa: F401
    from typing_extensions import TypeAlias

IP_Version: "TypeAlias" = "Literal[4, 6]"
```

EPEL 7 not having a `python3-typing-extensions` package is the reason for these Bad™ workarounds.